tools/tagger/tagger.js

```diff
@@ -1,55 +1,65 @@
-function sanitizeName(name) {
-  if (!name) {
-    return '';
-  }
-
-  const decodedText = decodeURIComponent(name.trim()).toLowerCase().normalize('NFD').replace(/[\u0300-\u036f]/g, '');
-  const sanitizedText = decodedText.split(',').map((text) => text.trim().replace(/[^a-z0-9]+/g, '-').replace(/^-|-$/g, '')).join(', ');
-  return sanitizedText.trim();
-}
-
 function renderItems(cat, catId, taxonomy) {
   let html = '';
-  const items = taxonomy.map((item) => item[cat]);
-  items.forEach((tag) => {
-    if (tag.trim() !== '') {
+  const oCategoryTags = taxonomy[cat];
+  oCategoryTags.forEach((tag) => {
+    if (tag.key.trim() !== '' && tag.text.trim() !== '') {
       html += `
       <span class="path">
-        <span data-title="${tag}" class="tag cat-${catId % 8}">${tag}</span>
+        <span data-title="${tag.key}" class="tag cat-${catId % 8}">${tag.text}</span>
       </span>
     `;
     }
   });
   return html;
 }
 
+const getReduceTags = (oTaxonomy, sCategory) => {
+  const sCategoryCapital = sCategory.charAt(0).toUpperCase() + sCategory.slice(1);
+
+  return oTaxonomy.reduce((oAccumulated, oObject) => {
+    // eslint-disable-next-line no-unused-expressions
+    oAccumulated[sCategory] || (oAccumulated[sCategory] = []);
+    oAccumulated[sCategory].push({
+      key: oObject[`${sCategoryCapital} Key`],
+      text: oObject[`${sCategoryCapital} Text`],
+    });
+    return oAccumulated;
+  }, {});
+};
+
 function initTaxonomy(taxonomy) {
   let html = '';
-  Object.keys(taxonomy[0]).forEach((cat, idx) => {
+  const subjectsTags = getReduceTags(taxonomy, 'subjects');
+  const industriesTags = getReduceTags(taxonomy, 'industries');
+  const tagsJson = { ...subjectsTags, ...industriesTags };
+
+  Object.keys(tagsJson).forEach((cat, idx) => {
+    const sCategory = cat.charAt(0).toUpperCase() + cat.slice(1);
     html += '<div class="category">';
-    html += `<h2>${cat}</h2>`;
-    html += renderItems(cat, idx, taxonomy);
+    html += `<h2>${sCategory}</h2>`;
+    html += renderItems(cat, idx, tagsJson);
     html += '</div>';
   });
+
   const results = document.getElementById('results');
   results.innerHTML = html;
 }
 
 async function getTaxonomy() {
-  const resp = await fetch('/tags.json');
+  const resp = await fetch('/tags-placeholders.json');
```

Comment:
can we avoid `placeholders` name? how about `new-tags` ?

Reply:
@ravkiran done renaming the files to new-tags.

cc: @jjjaspher